projects/BFDR.Tests/FetalDeathRecord_Should.cs

```diff
@@ -1253,6 +1253,83 @@ public void TestImportEducation()
       Assert.Equal("1", ije.MEDUC_BYPASS);
     }
 
+    [Fact]
+    public void TestParentReportedAge()
```

Comment:
It doesn't look like there's a test that confirms if the names are set in the incorrect order, an error is thrown. Add a test that confirms there's an error. There should be examples of similar tests in VRDR.Test

Reply:
I did not realize that was the case, but it makes sense.